build/docker/policies/API-Minimal.policy

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration>
+<policy>API Minimal</policy>
+<scanner>
+<level>MEDIUM</level>
+<strength>MEDIUM</strength>
+</scanner>
+<plugins>
+<p6>
+<enabled>false</enabled>
```

Comment:
Could be re-enabled per zaproxy/zap-extensions#898? (Seems to be ready for merge.)

EDIT: Moved the comment to the pull request.